docs/examples/one_tank.md

```diff
@@ -0,0 +1,219 @@
+# One-Tank Example
+
+This example shows how to train two different models to predict the time-series behavior of a single water tank with a height-dependent outflow and a time-varying inflow.
+It is inspired by the one-tank simulink model used for control design [^1].
+A schematic drawing of the system is shown in the following figure.
+
+![The one-tank system](./images/one_tank.svg)
+
+The complete source code for this example can be found [in the repository](https://github.com/flowcean/flowcean/blob/main/examples/one_tank/run.py).
+See also [Run this example](#run-this-example) on how to run this example locally.
+
+The dynamics of the system can be described by an ordinary differential equation (ODE).
+This type of equation relates the change in a variable (it's derivative) to its actual value.
+
+For this example the system can be descirbed by the ODE
+
+$$ \dot{x} = \frac{b V(t) - a \sqrt{x}}{A} $$
+
+where $x$ is the water level in the tank, $\dot{x}$ is the change of the water level over time, $V(t)$ is the time-dependent inflow, $A$ is the tank area, and $a$ and $b$ are scaling constants for the equation.
+The solution of an ODE is not a single value, but a function (here $x(t)$) or a series of it's values for different times $t$.
```

Comment:
Should we make this less general and only say how it is in our case. I guess, that would be more clear.

Reply:
Also for line 21

Reply:
I agree with the comment for line 21 and fixed it accordingly. I'd like to keep the statement from line 20 to emphasize the point that an ODE has a function as its solution, not a single value. This is probably clear to anyone who has worked with ODEs before, but not everyone has.